First pass HTML and CSS

reimerharts opened the pull request:
- Added the Roboto font.
- Added some section deviders.
- Changed some H tags for more SEO and easier customization.
- Added BG image.
- Got started on the CSS with the focus of mobile first.

reimerharts committed:
First pass HTML and CSS

- Added the Roboto font.
- Added some section deviders.
- Changed some H tags for more SEO and easier customization.
- Added BG image.
- Got started on the CSS with the focus of mobile first.

reimerharts committed:
Fonts uploaded for headers

- Uploaded fonts to use for headers on the characters

reimerharts committed:
Font visible for more browsers

Added .svg and .woff for a broader spectrum of browsers.

xathien commented on src/main/resources/templates/charsheet.html:
```diff
@@ -39,57 +41,53 @@
-          <span id="levelLabel" class="statLabel">LV</span>
-          <span id="levelValue" class="statValue" th:text="${character.level}" th:if="${!admin}">1</span>
+          <span id="levelLabel" class="statLabel"><p>LV</p></span>
+          <span id="levelValue" class="statValue" th:text="${character.level}" th:if="${!admin}"><p>1</p></span>
```
These `th:text` fields (used for Thymeleaf templating) will break with your new nested elements, as will all of the JS that updates these fields as you spend skill points or as the admin adjusts levels, etc., etc.

If you move the `th:text` and the `id` attributes into the nested `p` element for each of them, they Should™ continue working.

reimerharts replied:
Alright good to know!

xathien commented on src/main/resources/templates/charsheet.html:
```diff
@@ -39,57 +41,53 @@
-
       </div>
 
+      <div class="section_devider"></div>
```
```suggestion
      <div class="section_divider"></div>
```

xathien commented on src/main/resources/static/css/charsheet.css:
```diff
@@ -1,10 +1,46 @@
+  background-size: cover;
+}
+
+#main .section_devider {
```
```suggestion
#main .section_divider {
```

xathien reviewed (changes requested):
I'm also not seeing anywhere that you're using the new font files; am I missing something?

reimerharts committed:
Update src/main/resources/static/css/charsheet.css

Co-Authored-By: Zack Young <zyoung@jive.com>

reimerharts committed:
Update src/main/resources/templates/charsheet.html

Co-Authored-By: Zack Young <zyoung@jive.com>

reimerharts commented:
> I'm also not seeing anywhere that you're using the new font files; am I missing something?

Not yet. I didn't get to that in the first pass.

xathien commented on src/main/resources/static/css/charsheet.css:
```diff
@@ -13,65 +49,78 @@ body {
 }
 
-input.specName {
+input .specName {
```
Before, the CSS rule was "input with class `specName`". Now you have it as "input with a child with class `specName`", which won't ever evaluate.

(This comment applies to the 4 changes above this one, too)

reimerharts replied:
Got it. I don't remember why I changed that... 
Changed it back.

reimerharts committed:
Minor touchups and some reverted changes

Also started using the proper font for headers

reimerharts committed:
Merge branch 'feat/charsheet-css' of https://github.com/reimerharts/cyberscape-neo-discord-bot into feat/charsheet-css

reimerharts committed:
Updated skillCluster, skillLabel and skillValue

- Added background
- Added button image
- Added correct position to the skill value

xathien commented on src/main/resources/templates/charsheet.html:
```diff
@@ -15,21 +15,23 @@
-          th:href="@{/css/charsheet.css}" media="screen" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/charsheet.css"
+          th:href="@{charsheet.css}" media="screen" />
```
Let's make absolutely sure we put this back... 🙂 
```suggestion
          th:href="@{/css/charsheet.css}" media="screen" />
```

reimerharts committed:
Fire image removed, buttons beautified
